src/openeo_gfmap/utils/catalogue.py

```diff
@@ -204,18 +236,19 @@ def s1_area_per_orbitstate(
     }
 
 
-def select_S1_orbitstate(
+def select_s1_orbitstate_vvvh(
     backend: BackendContext,
     spatial_extent: SpatialContext,
     temporal_extent: TemporalContext,
 ) -> str:
     """Selects the orbit state that covers the most area of the given spatio-temporal context
-    for the Sentinel-1 collection.
+    for the Sentinel-1 collection. Only works for the product with the VV&VH polarisation.
 
     Parameters
     ----------
     backend : BackendContext
-        The backend to be within, as each backend might use different catalogues.
+        The backend to be within, as each backend might use different catalogues. Only the CDSE,
+        CDSE_STAGING and FED backends are supported.
```

Comment:
what makes it not supported for terrascope?

Reply:
Unfortunately CreoDIAS uses a non-conventional API to browse their S1/S2 catalogues (not implementing STAC specifications)

And because the collections on Terrascope for S1 are limited, I decided to focus this functionality for CDSE